engine/runtime/src/main/java/org/enso/interpreter/node/callable/InvokeCallableNode.java

```diff
@@ -375,8 +381,17 @@ static Object doPolyglot(
   @Fallback
   public Object invokeGeneric(
       Object callable, VirtualFrame callerFrame, State state, Object[] arguments) {
-    Atom error = EnsoContext.get(this).getBuiltins().error().makeNotInvokable(callable);
-    throw new PanicException(error, this);
+    if (isForOversaturatedArguments && schema.length >= 1 && schema[0].isNamed()) {
```

Comment:
Is this a compilation constant? Try `CompilerAsserts.partialEvaluationContant` or co.

Reply:
Tried in e645b0571576ef5b520287248c4b38537be3183d, is that how it is supposed to be used?